include/CL/cl_version.h

```diff
@@ -19,7 +19,8 @@
 
 /* Detect which version to target */
 #if !defined(CL_TARGET_OPENCL_VERSION)
-#define CL_TARGET_OPENCL_VERSION 120
+#pragma message("cl_version.h: CL_TARGET_OPENCL_VERSION is not defined. Defaulting to 300 (OpenCL 3.0)")
+#define CL_TARGET_OPENCL_VERSION 300
```

Comment:
Could you briefly note this particular change in the commit message, and why it won't break anything?

Reply:
Added in commit: Upgrade default version to opencl core 3.0. This will not break existing functio
nality because we are currently already building with 3.0 (see cMakeList). All f
uture version of opencl core is backward compatible.